package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@xroadsed/delta-client",
   "version": "0.1.7",
-  "description": "A minimal single-page application framework (specifically based on Eta).",
+  "description": "A minimal single-page application framework (specifically designed for Eta).",
```

Comment:
I agree, I just wanted to change the wording because it definitely was not based on Eta. It was designed for Eta, at least originally. If we want to change that in the future, I don't have a problem with it; but this PR is not the place to discuss it, and it's not something I want to spend time on until Element's deployable.